### PR TITLE
Amazon ECS Agent metrics

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Updates the Amazon ECS Agent data collected docs to forward users to the Docker integration metrics section

### Motivation
ECS metrics come from the Docker integration and are tagged accordingly with ECS task tags

### Preview
https://docs-staging.datadoghq.com/sarina/ecs-agent-docs/agent/amazon_ecs/data_collected/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/agent/amazon_ecs/data_collected.md [hunk 1]
```diff
@@ -17,24 +17,21 @@ further_reading:
 
 ### Metrics
 
-Amazon ECS metrics collected with the Datadog Agent:
+Amazon ECS on EC2 is a container management service for Docker containers running on EC2 instances. Metrics collected by the Agent when deployed in a Docker container are the same metrics collected by the Docker integration. Refer to the [Docker integration metrics][1] for a complete list of metrics.
```
Comment (cohenyair) [inline content/en/agent/amazon_ecs/data_collected.md:20]: can we also mention that the Docker metrics will have these following four tags:
https://share.getcloudapp.com/4guJPEOZ

Comment (cohenyair) [reply content/en/agent/amazon_ecs/data_collected.md:20]: just noticed the note below, let's just add the names of the tags please

**Review by cohenyair — CHANGES_REQUESTED** [review]

File: content/en/agent/amazon_ecs/data_collected.md (hunk 1 above)
Comment (sarina-dd) [reply content/en/agent/amazon_ecs/data_collected.md:20]: There are five tags in the screenshot, should I list all? Or just the first four?

**Comment by sarina-dd** [comment]
@cohenyair Added the tags 👍

**Comment by sarina-dd** [comment]
@cohenyair Can we get a 👍  so we can push the changes?

**Review by kayayarai — APPROVED** [review]
Suggestion: change the word "whitelist," but otherwise 👍

**Pull request merged**